src/sentry/static/sentry/app/utils/convertFromSelect2Choices.tsx

```diff
@@ -1,5 +1,11 @@
-// Converts arg from a `select2` choices array to a `react-select` `options` array
-const convertFromSelect2Choices = choices => {
+import {SelectValue} from 'app/types';
+
+type Choices = string[] | [string, string][] | any;
```

Comment:
Maybe you could add a `TODO (ts)` here, to remember that we have to replace `any`  with the missing types 🙂